core/arch/arm/arm.mk

```diff
@@ -245,6 +245,7 @@ ta-mk-file-export-add-ta_arm32 += CROSS_COMPILE32 ?= $$(CROSS_COMPILE)_nl_
 ta-mk-file-export-add-ta_arm32 += CROSS_COMPILE_ta_arm32 ?= $$(CROSS_COMPILE32)_nl_
 ta-mk-file-export-add-ta_arm32 += COMPILER ?= gcc_nl_
 ta-mk-file-export-add-ta_arm32 += COMPILER_ta_arm32 ?= $$(COMPILER)_nl_
+ta-mk-file-export-add-ta_arm32 += PYTHON3 ?= $(PYTHON3)_nl_
```

Comment:
Sorry, I'm having second thoughts here. This is OK assuming we want to make the value *used when building the OP-TEE OS* be the *default* value used to build TAs. But why would we want to do that? The two build environments may be different. Using the same default for both (i.e., `python3`) would make more sense I suppose. Like so:

```
ta-mk-file-export-add-ta_arm32 += PYTHON3 ?= python3_nl_
```

Reply:
Ok, that makes sense. Pushed a new revision.